[Misc] Added Code of Conduct to repository. (#220)
Despair-Games/poketernity: README.md

# Contributing

> Please take a look at our Contributor Code of Conduct [here](./CODE_OF_CONDUCT.md).

Reviewer: This is not a "nice to have".
This is a requirement!

```suggestion
Make sure to read our [Code of Conduct](./CODE_OF_CONDUCT.md) before contributing!
```
